### DISCUSSION TITLE
app.bsky.graph.listblock

### DISCUSSION THREAD

**Discussion by seboslaw** [bluesky-social/atproto#1686] [Clarifications and "Why?"] [answered]
The description "A block of an entire list of actors" of this one is a little ambiguous. Am I right to assume that this will let me block all actors from a (mute) list at once?

**Comment by devinivy** [comment, marked as answer]
Yep, exactly.  Creating a `app.bsky.graph.block` record allows you to block a single actor, and `app.bsky.graph.listblock` allows you to block a list of actors.  The list-block is also dynamic: if an actor is added to the list after you create the `app.bsky.graph.listblock` record, you will start blocking them.

**Reply by seboslaw** [reply]
@devinivy worked like a charm. Added it to my app today.
